### PR TITLE
New Post: Add-plugins, Add-mods

### PR DESCRIPTION
**Pull request by theTWIXhunter**

### CODE REVIEW

File: 2024-07-10-install-mods.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: How to add Mods to your server
+category: Modifications
+tags: Mods
+permalink: minecraft/modifications/mods/add-mods
+description: Learn how to upload and install Mods to your Minecraft Java server.
+author: The_twix_hunter
```
Comment (iamdeka) [inline 2024-07-10-install-mods.md:8]: `theTWIXhunter` looks better

File: _drafts/minecraft/plugins/2024-07-09-install-plugins.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
+---
+layout: post
+title: How to add plugins to your server
+category: Modifications
+tags: Plugins
+permalink: minecraft/modifications/plugins/add-plugins
+description: Learn how to upload and install plugins to your Minecraft Java server.
+author: The_twix_hunter
```
Comment (iamdeka) [inline _drafts/minecraft/plugins/2024-07-09-install-plugins.md:8]: `theTWIXhunter` looks better

**Review by iamdeka — CHANGES_REQUESTED** [review]

File: 2024-07-10-install-mods.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    - [Modrinth](https://modrinth.com/mods)
+
+{: .warning}
+
+> Before downloading, take a look at:
+> - the Minecraft version of the mod
+> - the modloader needed (any forge Mod will work on neoforge servers)
+> and make sure those are the same as your server
+
+4. Navigate to the [FileManager](https://client.falixnodes.net/server/filemanager?dir=/) by hovering over "Manage" and clicking ["FileManager"](https://client.falixnodes.net/server/filemanager?dir=/) or open your SFTP program like FileZilla.
```
Comment (iamdeka) [inline 2024-07-10-install-mods.md:43]: Please, don't write File Manager like that. Hyperlinks can have spaces.

**Review by iamdeka — APPROVED** [review]

**Comment by iamdeka** [comment]
You also put the articles in the wrong folders
They should be in `/content/_posts`

**Comment by iamdeka** [comment]
The dates and file names for linking are also mismatching. Please pay more attention to where you're putting the articles, and how they are named

File: 2024-07-10-install-mods.md (hunk 1 above)
Comment (theTWIXhunter) [reply 2024-07-10-install-mods.md]: Oh, I will note (and change) that Will

**Review by iamdeka — APPROVED** [review]

**Comment by theTWIXhunter** [comment]
wait, I commited the old files again

**Comment by theTWIXhunter** [comment]
whoeps wrong file

**Comment by theTWIXhunter** [comment]
ok, recommited your changes

**Review by iamdeka — CHANGES_REQUESTED** [review]
...

dGz14o0Q0V

**Comment by theTWIXhunter** [comment]
eurgg

**Comment by theTWIXhunter** [comment]
I don't have a mc/java in my main

**Comment by theTWIXhunter** [comment]
fixed it (hope fullt)

**Comment by iamdeka** [comment]
> fixed it (hope fullt)

I don't see any commits

**Comment by theTWIXhunter** [comment]
> I don't see any commits

I committed it

**Comment by theTWIXhunter** [comment]
Committed is a small word, I committed it 6 times (I moved them to Java/mods and Java/plugins and I renamed them to 2024-07-27 (I made them on 10 that’s why they where called that on my pc)

**Comment by iamdeka** [comment]
m0E9xVMdFX

Put them in /content/....

**Review by iamdeka — CHANGES_REQUESTED** [review]
⬆️

**Comment by theTWIXhunter** [comment]
looks like they are now in /content/

**Review by iamdeka — APPROVED** [review]

**Comment by theTWIXhunter** [comment]
I didn't close this

**Comment by iamdeka** [comment]
> deleted the branch FalixNodes-Software:main

🤦

**Comment by KuroiJigoku** [comment]
did bro just- deleted the branch that had these stuffs 🤦

**Comment by theTWIXhunter** [comment]
> did bro just- deleted the branch that had these stuffs 🤦

I messed up some articles between diffrt Branches so I just reuploaded to a new branch

**Pull request closed without merging**